frontend/src/api.ts

```diff
@@ -457,18 +457,18 @@ export const api = {
   async deleteFeedbackReply(feedbackId: number, replyId: number) {
     return apiV2.delete<unknown, void>(`/feedbacks/${feedbackId}/replies/${replyId}`);
   },
-  async runAdHocTest(projectId: number, testUuid: string, inputs: Array<FunctionInputDTO>) {
-    return apiV2.post<unknown, TestTemplateExecutionResultDTO>(`/testing/tests/run-test`, {
+  async runAdHocTest(projectId: number, testUuid: string, inputs: Array<FunctionInputDTO>, sample: boolean) {
+    return apiV2.post<unknown, TestTemplateExecutionResultDTO>(`/testing/tests/run-test?sample=${sample}`, {
       projectId,
       testUuid,
-      inputs,
+      inputs
     });
   },
   async getCatalog(projectId: number) {
     return apiV2.get<unknown, CatalogDTO>(`/catalog`, {
       params: {
-        projectId,
-      },
+        projectId
+      }
     });
   },
```

Comment:
Why don't we use `projectId` as a path param?

Reply:
It's true that it also can be a path variable in this case. I think it was set as a filter to show the data associated to the project but now that all catalog items will be linked to projects it make more sense to use a path variable